docs/preview/continuous-patching/README.md

```diff
@@ -0,0 +1,468 @@
+Continuous Patching Workflow in Azure Container Registry
+========================================================
+
+## Introduction
+
+Continuous Patching is a feature in Azure Container Registry that allows you to recurringly scan and patch specified artifacts for only OS-level  vulnerabilities within the registry. The workflow enables to set a cadence for run and create a recurring ACR task that scans your configured list of images for vulnerabilities (CVEs) using [Trivy](https://trivy.dev/) and patch them using [Copa](https://project-copacetic.github.io/copacetic/website/).
```

Comment:
Azure Container Registry Continuous Patching allows you to recurringly scan and patch registry artifacts and container images that have OS-level vulnerabilities. Continuous Patching is able to patch vulnerable container images within the registry, without needing access to the original source code or build pipeline of the container image.